src/debugger/debugger.c

```diff
@@ -406,6 +408,7 @@ void xdebug_debugger_throw_exception_hook(zend_object *exception, zval *file, zv
 		}
 #endif
 		if (exception_breakpoint_found && xdebug_handle_hit_value(extra_brk_info)) {
+			xdebug_enable_debugger_and_rebuild_stack_if_disabled();
```

Comment:
When starting the debugger on an exception breakpoint, we need to enable it and rebuild the stack if needed